docs/deployment/feature_gates.md

```diff
@@ -29,7 +29,8 @@ The following tables are a summary of the feature gates that you can set on diff
 | NodeAgentAuthorizer                      | `false` | `Alpha` | `1.109` |         |
 | CredentialsRotationWithoutWorkersRollout | `false` | `Alpha` | `1.112` |         |
 | InPlaceNodeUpdates                       | `false` | `Alpha` | `1.113` |         |
-| RemoveAPIServerProxyLegacyPort           | `false` | `Alpha` | `1.113` |         |
+| RemoveAPIServerProxyLegacyPort           | `false` | `Alpha` | `1.113` | `1.115` |
+| RemoveAPIServerProxyLegacyPort           | `false` | `Beta`  | `1.116` |         |
```

Comment:
In case you decide to enable it by default.

```suggestion
| RemoveAPIServerProxyLegacyPort           | `true`  | `Beta`  | `1.116` |         |
```